src/best-practices/general/magento-commerce-cloud-best-practice-for-slave-connection.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Adobe Commerce on cloud architecture best practice for slave connection
+title: Adobe Commerce on cloud architecture best practice for MySQL slave connection
```

Comment:
```suggestion
title: Adobe Commerce on our cloud infrastructure best practice for MySQL slave connection
```

Reply:
I know we used architecture ourselves, but there was a twist and it will be "on our cloud infrastructure" (and "our" is required).